scripts/upload/add_ssa.py

```diff
@@ -1,36 +1,40 @@
+#!/usr/bin/env python
+
 """
-Added ssa measurements to the database.
-1. Data must be downloaded via sh ../download/download_nsidc.sh
-2A. python run.py # To run all together all at once
-2B. python add_ssa.py # To run individually
+Added SSA measurements from:
+* 2020
 """
 
-import glob
-from os.path import abspath, join
-
-from snowex_db.batch import UploadProfileBatch
-
-
-def main():
+from import_logger import get_logger
+from earthaccess_data import get_files
+from snowexsql.db import db_session_with_credentials
+from snowex_db.upload.layers import UploadProfileData
 
-    # Obtain a list of SSA profiles
-    directory = abspath('../download/data/SNOWEX/SNEX20_SSA.001/')
-    filenames = glob.glob(join(directory, '*/*.csv'))
+LOG = get_logger()
 
-    # Instantiate the uploader
-    b = UploadProfileBatch(
-        filenames,
-        debug=False,
-        doi="https://doi.org/10.5067/SNMM6NGGKWIT",
-        in_timezone='MST'
-    )
+# Map of DATA SET ID to DOI from NSIDC
+# * https://nsidc.org/data/snex20_ssa/versions/1
+SSA_DOI = {
+    "SNEX20_SSA": "10.5067/SNMM6NGGKWIT",
+    # "SNEX23_SSA": "10.5067/BSEP59ADC6XN",
+    # "SNEX23_SSA_SO": "10.5067/9SY1H2L0BY0X",
+    # "SNEX23_OCT22_SSA" : "10.5067/CPQ2DA73IZVH",
```

Comment:
These are all published SSA datasets. Unfortunately (and of course) they changed the CSV format.